src/index.js

```diff
@@ -7,6 +7,9 @@ import {css, styleSheet} from 'glamor'
 import shouldForwardProperty from './should-forward-property'
 
 const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+const domElements = htmlTagNames.concat(svgTagNames)
```

Comment:
We could actually do that dedupe in the `Object.assign` calls below:

```javascript
Object.assign(
  glamorous,
  domElements.reduce(
    (getters, tag) => {
      if (getters[tag]) { // <-- this prevents overwriting
        return getters
      }
      getters[tag] = glamorous(tag)
      return getters
    },
    {},
  ),
)

// ...

Object.assign(
  glamorous,
  domElements.reduce(
    (comps, tag) => {
      const capitalTag = capitalize(tag)
      if (comps[capitalTag]) { // <-- this prevents overwriting
        return comps
      }
      comps[capitalTag] = glamorous[tag]()
      comps[capitalTag].displayName = `glamorous.${capitalTag}`
      comps[capitalTag].propsAreCssOverrides = true
      return comps
    },
    {},
  ),
)
```

Alternatively we could just do:

```javascript
domElements = htmlTagNames.concat(svgTagNames).filter((tag, index, array) => array.indexOf(tag) === index)
```

I think that should work :)